<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="0" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="com.automation.tests.TestWeatherAPI" duration-ms="1274" started-at="2020-06-09T00:51:13Z" finished-at="2020-06-09T00:51:14Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="1274" started-at="2020-06-09T00:51:13Z" finished-at="2020-06-09T00:51:14Z">
      <class name="com.automation.tests.TestWeatherAPI">
        <test-method status="SKIP" signature="registerWeatherStationThroughPostCall(com.automation.model.BeanClassData)[pri:0, instance:com.automation.tests.TestWeatherAPI@17d677df]" name="registerWeatherStationThroughPostCall" duration-ms="0" started-at="2020-06-08T19:51:13Z" data-provider="RegisterWeatherStations" finished-at="2020-06-08T19:51:13Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method public void com.automation.tests.TestWeatherAPI.registerWeatherStationThroughPostCall(com.automation.model.BeanClassData) requires a @DataProvider named : RegisterWeatherStations in class com.automation.utils.DataProviders]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method public void com.automation.tests.TestWeatherAPI.registerWeatherStationThroughPostCall(com.automation.model.BeanClassData) requires a @DataProvider named : RegisterWeatherStations in class com.automation.utils.DataProviders
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:262)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:418)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:992)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1082)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registerWeatherStationThroughPostCall -->
        <test-method status="SKIP" signature="testDataProviderUsingJsonExample(com.automation.model.BeanClassData)[pri:0, instance:com.automation.tests.TestWeatherAPI@17d677df]" name="testDataProviderUsingJsonExample" duration-ms="0" started-at="2020-06-08T19:51:13Z" data-provider="newJson" finished-at="2020-06-08T19:51:13Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[Can't invoke method public java.lang.Object[][] com.automation.utils.DataProviders.getData() throws java.io.FileNotFoundException on this instance of class com.automation.tests.TestWeatherAPI due to class mismatch]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: Can't invoke method public java.lang.Object[][] com.automation.utils.DataProviders.getData() throws java.io.FileNotFoundException on this instance of class com.automation.tests.TestWeatherAPI due to class mismatch
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:73)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:136)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:992)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1082)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDataProviderUsingJsonExample -->
        <test-method status="PASS" signature="weatherStationWithoutAPIKey()[pri:0, instance:com.automation.tests.TestWeatherAPI@17d677df]" name="weatherStationWithoutAPIKey" duration-ms="1263" started-at="2020-06-08T19:51:13Z" finished-at="2020-06-08T19:51:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- weatherStationWithoutAPIKey -->
      </class> <!-- com.automation.tests.TestWeatherAPI -->
    </test> <!-- Command line test -->
  </suite> <!-- com.automation.tests.TestWeatherAPI -->
</testng-results>
